src/actions/actionZindex.tsx

```diff
@@ -133,13 +134,14 @@ export const actionSendToBack: Action = {
   },
   contextItemLabel: "labels.sendToBack",
   commitToHistory: () => true,
-  keyTest: event => event[KEYS.META] && event.shiftKey && event.key === "B",
+  keyTest: event =>
+    event[KEYS.CTRL_OR_CMD] && event.altKey && event.key === "[",
```

Comment:
My bad, this one is not working.

Reply:
Right, this is our priority system in work (or, more like the fact we have no priority system in place). It matches `event[KEYS.CTRL_OR_CMD] && event,key === "["` from above, and exits early. One short-term solution is to move this key above it so it matches first.

Reply:
Didn't work.. Feel free to commit directly into this branch if you know how to solve this.. I

Reply:
I've just pushed what you've definitely tried, but for me it works :man_shrugging:

Reply:
`Cmd+Option+[`, `Cmd+Option+]`

Don't work for me.. Mac with Chrome, Safari with US layout keyboard... very strange

Reply:
Mhm.. `Option` is still `event.altKey`, right?

Reply:
I'm not sure what behavior you're seeing exactly. Is it matching the wrong hotkey, or not matching any at all? What `event.key` and modifiers are shown in the `index.tsx` `onKeyDown` handler?

Reply:
Just doesn't bring it to front nor back with that shortcut.. I'm guessing you are not on a Mac..

Reply:
Nope, Windows. So it doesn't do anything... I'd debug the `KeyboardEvent` directly.

Reply:
`Ctrl+Alt+[]` works for you on Windows?

Reply:
Yep :/.